src/service/performance-impl.js

```diff
@@ -415,6 +425,14 @@ export class Performance {
       layoutInstabilityObserver.observe({type: 'layout-shift', buffered: true});
     }
 
+    if (this.supportsLargestContentfulPaint_) {
+      const lcpObserver = new this.win.PerformanceObserver(list => {
+        list.getEntries().forEach(processEntry);
+        this.flush();
+      });
```

Comment:
Another annoying comment: can we de-dupe the creation of these?

```js
const lcpObserver = this.createPerformanceObserver_();

/** 
 * @param {function(!PerformanceEntry)} processEntry
 * @return {!PerformanceObserver}
 * @private
 */
createPerformanceObserver_(processEntry) {
  return new this.win.PerformanceObserver(list => {
    list.getEntries().forEach(processEntry);
    this.flush();
  });
}
```

Reply:
Done in efad989cd11bd421fc04b1b77c7b076ae3edd5a7